### PR TITLE
feat: added new update system

### PR DESCRIPTION
**Pull request by gerblesh**
Discussion: https://github.com/orgs/ublue-os/discussions/187
Solves: https://github.com/ublue-os/config/issues/63

after quite a bit of troubleshooting/fixing i've finally gotten the ublue-updater to build, and made a basic draft pull request to add the new update system.
This is by no means meant to be merged right away, and will likely need more changes/review (particularly moving the repo into the uBlue org)

Currently in the PR:
remove the old update services and update services RPM
pull in new update system from https://github.com/gerblesh/ublue-updater

### CODE REVIEW

**Comment by gerblesh** [comment]
Update: basically everything works that needs to work as an MVP, updating, notification, logging failed updates, etc. It's already quite a bit more polished than the current update system we have in place. I'd say we just move the repo into the uBlue org and merge, that being said, I think it would be good to get some acknowledgement from image maintainers to make sure they are on board with this before merging

**Review by marcoceppi — CHANGES_REQUESTED** [review]
Going to put a pause on this after the discussions in discord. The trend is that this is something better suited for bling and not as the default for main and those "upstream of custom" images.

**Comment by gerblesh** [comment]
Closing this as it's out of scope for the uBlue main images and config, will make a PR in https://github.com/ublue-os/bling instead

**Pull request closed without merging**
